Orange/util.py

```diff
@@ -3,20 +3,25 @@
 from functools import wraps
 from itertools import chain, count
 from collections import OrderedDict
-import logging
+import warnings
 
 import numpy as np
 
 
-log = logging.getLogger()
+warnings.simplefilter('default', DeprecationWarning)
```

Comment:
IIUC, this should make e.g. below `DeprecationWarning` fire only once per each line where the deprecated function is used. Perhaps there is a more canonical place to install this filter instead of this module?